java/query/src/test/java/sleeper/query/lambda/SqsQueryProcessorLambdaIT.java

```diff
@@ -211,7 +214,9 @@ public void shouldSplitUpQueryWhenItSpansMultiplePartitions() throws Exception {
                 .queryId("abc")
                 .regions(List.of(new Region(List.of(range1, range2, range3))))
                 .build();
+
         processQuery(query);
+        processLeafPartitionQuery();
```

Comment:
It doesn't seem relevant to this test to actually process a leaf partition query - can we keep this test asserting on the state after the query is split but before any leaf partition queries are processed?

It does seem useful to have a test for the state where only one of the leaf partition queries have been processed though, can we make that a separate test?